packages/solid/src/server/rendering.ts

```diff
@@ -68,6 +68,18 @@ export function createUniqueId(): string {
   return `${ctx.id}${ctx.count++}`;
 }
 
+export function createServerValue<T>(cb: () => T): T {
+  const ctx = sharedConfig.context;
+  // This might be fun specially if the value is a Promise
+  // but of course, the only way they can translate the Promise
```

Comment:
what do you mean by "translte the Promise"

Reply:
convert the Promise into something the components and the reactivity system can understand